src/components/organisms/EarthEditor/CanvasArea/hooks.ts

```diff
@@ -140,6 +149,39 @@ export default (isBuilt?: boolean) => {
     document.title = title;
   }, [isBuilt, title]);
 
+  const [updateLayerLatLng] = useUpdatePropertyValueLatLngMutation();
+
+  const handleDragLayer = useCallback((_layerId: string, _position: LatLngHeight | undefined) => {
+    setIsDragging(true);
+  }, []);
+
+  const handleDropLayer = useCallback(
+    async (layerId: string, position: LatLngHeight | undefined) => {
+      setIsDragging(false);
+      const layerProperty = layers?.layers.find(l => l.id === layerId)?.rawProperty;
+      const fieldId = "location";
+      const schemaItemId = "default";
```

Comment:
I feel like this callback is noisier than it needs to be. Can't we just set these in the updateLayerLatLng call? Rather than here writing `const fieldId = "location"`, below we can just pass it and schemaItemId in like `fieldId: "location"`. 
Again, my understanding is to do what you did only for clarity, which the param names updateLayerLatLng is expecting are clear enough (the exact same actually).